Views/InputsView.swift

```diff
@@ -321,23 +323,111 @@ class InputsView: UIView, UITableViewDataSource, UITableViewDelegate, UISearchBa
     return true
   }
 
-  private func loadSampleInputs() {
-    createSeed(registered: true, variety: "Variété 1", specie: "Espèce 1")
-    createSeed(registered: true, variety: "Variété 2", specie: "Espèce 2")
+  private func loadRegisteredInputs() {
+    createSeed(variety: "Variété 1", specie: "Espèce 1")
```

Comment:
Why do we still have this instead of having the same thing as we have for the Phyto products and the Fertilizers?